### PR TITLE
📖 update build badges

### PR DESCRIPTION
**Pull request by tuminoid**
Update build badges for 0.5 branch to show 1.6 integration job results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ development. Here are the tests which run on different branches in BMO:
 * **/test-ubuntu-integration-main** runs ansible integration tests with CAPM3
   main branch and BMO **main** branch on Ubuntu
 
+**Release-0.5 branch:**
+
+* **/test-centos-e2e-integration-release-1-6** runs CAPM3 e2e integration tests
+  with CAPM3 release-1.5 branch and BMO **release-0.5** branch on Centos
```
Comment (smoshiur1237) [inline README.md:52]:
```suggestion
  with CAPM3 release-1.6 branch and BMO **release-0.5** branch on Centos
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ development. Here are the tests which run on different branches in BMO:
...
+* **/test-ubuntu-integration-release-1-6** runs ansible integration tests with
+  CAPM3 release-1.5 branch and BMO **release-0.5** branch on Ubuntu
```
Comment (smoshiur1237) [inline README.md:54]:
```suggestion
  CAPM3 release-1.6 branch and BMO **release-0.5** branch on Ubuntu
```

**Review by smoshiur1237 — COMMENTED** [review]
Same here

File: README.md (hunk 1 above)
Comment (tuminoid) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (tuminoid) [reply README.md]: Fixed

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-main test-centos-e2e-integration-main metal3-bmo-e2e-test
Documentation change.

**Comment by tuminoid** [comment]
/override test-ubuntu-integration-release-1-6 test-centos-e2e-integration-release-1-6 metal3-bmo-e2e-test

**Pull request merged**